tascrum/tests/test_card.py

```diff
@@ -273,16 +273,16 @@ def test_Create_Card_POST(self):
     #     self.assertIsNotNone(new_card)
 
     
-    # def test_card_count(self):
-    #     self.authenticate()
-    #     card_data = {"title":"card test","list":"1","startdate":'2022-05-15',"duedate":'2024-05-15',"reminder":'1 Day before'}
-    #     response = self.client.post(reverse('crcard-list') , card_data)
-    #     self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+    def test_card_count(self):
+        self.authenticate()
+        card_data = {"title":"card test","list":"1","startdate":'2022-05-15',"duedate":'2024-05-15',"reminder":'1 Day before'}
+        response = self.client.post(reverse('crcard-list') , card_data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
this part could be checked for giving bad_request to.

Reply:
it is handled.